driver-core/src/test/java/com/datastax/driver/core/CCMAccess.java

```diff
@@ -236,4 +243,14 @@
      */
     void waitForDown(int node);
 
+    /**
+     * @return The desired target protocol version based on the 'cassandra.version' System property.
+     */
+    ProtocolVersion getDesiredProtocolVersion();
```

Comment:
Ah I see that the documentation doesn't completely reflect that, i'll fix that up:

```
 The target protocol version based on the 'cassandra.version' System property.
```